src/Elm/Kernel/Utils.js

```diff
@@ -63,6 +63,20 @@ function _Utils_eqHelp(x, y, depth, stack)
 		y = __Dict_toList(y);
 	}
 	//*/
+	
+	if (typeof DataView === "function" && x instanceof DataView) {
+		let length = x.byteLength;
```

Comment:
`let` would cause a syntax error in Internet Explorer, right? So should be `var`? Same thing in the for-loop below.

Reply:
Yes of course! I will fix this. IE 😠

Reply:
Taken me 6 months!